org.adempiere.base/src/org/compiere/model/GridTable.java

```diff
@@ -3015,7 +3015,9 @@ public void dataRefreshAll(boolean fireStatusEvent, int rowToRetained)
 			// String whereClause = m_whereClause;

```

Comment:
Hi @igorpojzl - in my suggested patch I removed the two comment lines 3015 and 3023.

I think is better to remove those lines as they are misleading and inserting a problem when the comment is removed.

Regards,

Carlos Ruiz

Reply:
Commited Changes